app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -133,14 +135,18 @@ private fun MessageComposer(
             ) {
                 Box(
                     Modifier
-                        .weight(1f)
-                        .clickable(
-                            indication = null,
-                            interactionSource = remember { MutableInteractionSource() }
-                        ) {
-                            focusManager.clearFocus()
-                            messageComposerState.clickOutSideMessageComposer()
+                        .pointerInput(Unit) {
+                            detectTapGestures(
+                                onPress = {
+                                    focusManager.clearFocus()
+                                    messageComposerState.clickOutSideMessageComposer()
+                                },
+                                onDoubleTap = { /* Called on Double Tap */ },
+                                onLongPress = { /* Called on Long Press */ },
```

Comment:
Is this the box containing the whole recycler view of messages? If so, can we extract it to a separate component and/or add some documentation explaining it? otherwise it is quite difficult to follow what is what on a long nested list of composables without name or descriptions 🙈